public/base-locales/de.json

```diff
@@ -455,18 +455,18 @@
   "error_creating_user": "Neuer Benutzer kann derzeit nicht registriert werden",
   "discord_linked": "Discord Verknüpft",
   "telegram_linked": "Telegram Verknüpft",
-  "slider_little": "Kleine Tasse",
-  "tile_servers": "Kachelserver",
-  "tile_servers_default": "Standard",
+  "slider_little": "kleine Schieberegler",
+  "tile_servers": "Tileserver",
+  "tile_servers_default": "Standard Tileserver",
   "alola": "Alola",
-  "galarian": "Galarian",
+  "galarian": "Galar",
   "popups": "Popups",
-  "submission_cells_subtitle": "Zeigt nützliche Informationen zum Einreichen neuer POIs an",
+  "submission_cells_subtitle": "Zeigt nützliche Informationen zum Einreichen neuer POIs an.",
   "my_location": "Mein Standort",
   "server_dev_error_0": "{{variable_0}}",
   "navigation_controls": "Navigationssteuerung",
-  "slot_changes_poracle": "slot_changes",
-  "level": "eben",
+  "slot_changes_poracle": "Platzänderungen",
```

Comment:
pls use singular form, singular form is also used in poracle translations

Reply:
fixed